### ISSUE TITLE
Emails Not Being Sent to Agents

### ISSUE THREAD

**Issue by profiphotos** [ThemeAvenue/Awesome-Support#56]
Has no one else noticed this.  When a ticket is created, a support agent is assigned to the ticket and I'm guessing is supposed to get an Email telling him/her that they have a support issue to deal with.  Well no Email gets sent to the agent.  Even though there is a template for that in the Settings area.

Is this a bug?  Has no one else noticed this?

Cheers,
Markus

**Comment by julien731** [comment]
The tricky part with e-mail, when sent with PHP, is that sending the e-mail and receiving the e-mail are two very different things. Are you sure that the e-mail is NOT sent? Because it is very possible that the e-mail is sent even though you didn't receive it.

What I would advise you to test this is to use a real mail server to send the e-mails from WordPress. You can do that by installing a simple SMTP plugin like [WP Mail SMTP](https://wordpress.org/plugins/wp-mail-smtp/) or you can go for a more advanced solution like [Mandrill](http://mandrill.com/) (they have a WordPress plugin).

**Comment by profiphotos** [comment]
Thanks for the suggestions!  I've now got Mandrill installed and working.  I logged into the site as a regular user and placed a ticket.  Everything went fine ... I received the E-Mail saying that I had send a ticket, however, the Support Agent assigned to the ticket was never sent an Email notification.

It appears to be a bug, because I couldn't find any other setting that I had to turn on for this to happen.

Cheers,
Markus
